core/sqf/src/seatrans/tm/hbasetmlib2/javaobjectinterfacetm.cpp

```diff
@@ -428,6 +428,42 @@ bool  JavaObjectInterfaceTM::getExceptionDetails(JNIEnv *jenv)
    return true;
 }
 
```

Comment:
It would be better use the existing getExceptionDetails function to avoid duplication of code. You can either to choose to have default parameter or make the existing getExceptionDetails to call the new functions with retCode passed in